src/custom/pages/App/index.tsx

```diff
@@ -11,10 +11,19 @@ import About from 'pages/About'
 import Rewards from 'pages/Rewards'
 import Faq from 'pages/Faq'
 import CowGame from 'pages/CowGame'
+import useParseReferralQueryParam from 'hooks/useParseReferralQueryParam'
 
 export const Wrapper = styled(AppMod)``
 
 export default function App() {
+  const referralAddress = useParseReferralQueryParam()
+
+  useEffect(() => {
+    if (referralAddress) {
+      console.log('Referral address', referralAddress)
```

Comment:
Looks like useless code?

Reply:
Not really, this can updated later when we have the redux boilerplate in place to update the app state with the parsed value